extensions/amp-story/1.0/amp-story.js

```diff
@@ -1487,19 +1505,59 @@ export class AmpStory extends AMP.BaseElement {
       if (this.sidebar_ && sidebarState) {
         this.sidebarObserver_.observe(this.sidebar_, {attributes: true});
         actions.execute(this.sidebar_, 'open', /* args */ null,
-        /* source */ null, /* caller */ null, /* event */ null,
+            /* source */ null, /* caller */ null, /* event */ null,
             ActionTrust.HIGH);
+        this.openOpacityMask_();
       } else {
         this.sidebarObserver_.disconnect();
       }
     } else if (this.sidebar_ && sidebarState) {
       actions.execute(this.sidebar_, 'open', /* args */ null,
           /* source */ null, /* caller */ null, /* event */ null,
           ActionTrust.HIGH);
+      this.openOpacityMask_();
       this.storeService_.dispatch(Action.TOGGLE_SIDEBAR, false);
     }
   }
 
+  /**
+   * @private
+   */
+  openOpacityMask_() {
+    this.mutateElement(() => {
```

Comment:
Nit: only the parts where you write in the DOM should be wrapped in this `mutateElement`: the classList.add/remove, and the appendChild